src/style.css

```diff
@@ -1 +1,83 @@
-/* Styles go here */
+body {
+  font-family: "Roboto", "Arial", sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  height: 60px;
```

Comment:
set height for the nav tag and use line-height: 60px for centered text

Reply:
I wanted to make the clickable area bigger due to the height